pages/entity/_type/_.vue

```diff
@@ -39,52 +71,123 @@
   import axios from 'axios';
 
   import AlertMessage from '../../../components/generic/AlertMessage';
+  import InfoMessage from '../../../components/generic/InfoMessage';
   import BrowseChip from '../../../components/browse/BrowseChip';
+  import SearchResultsList from '../../../components/search/SearchResultsList';
+  import PaginationNav from '../../../components/generic/PaginationNav';
 
-  import getEntity, { getEntityPath, relatedEntities } from '../../../plugins/europeana/entity';
+  import * as entities from '../../../plugins/europeana/entity';
+  import search, { pageFromQuery } from '../../../plugins/europeana/search';
 
   export default {
     components: {
       AlertMessage,
-      BrowseChip
+      InfoMessage,
+      BrowseChip,
+      SearchResultsList,
+      PaginationNav
+    },
+    props: {
+      perPage: {
+        type: Number,
+        default: 24
+      }
     },
     data () {
       return {
         error: null,
         title: null,
-        relatedEntities: null
+        entity: null,
+        relatedEntities: null,
+        searchResults: {
+          error: null,
+          errorNoResults: 'No results',
```

Comment:
In the locale file.

Reply:
actually it's not used here, I've removed it and moved the one in the search page(where this was copied from) into the locale file.